build/images/perftool/README.md

```diff
@@ -1,14 +1,21 @@
 # images/perftool
 
-This Docker image is a very lightweight image based on Ubuntu 20.04 which
-includes the apache2-utils and iperf3 packages.
+This Docker image is for performance testing with iperf3. For Linux, it is
+based on Ubuntu 20.04 which includes the apache2-utils and iperf3 packages.
+For Windows, it downloads iperf3 binary and adds it to PATH.
 
 If you need to build a new version of the image and push it to Dockerhub, you
 can run the following:
 
-```bash
+```bash (Linux)
 cd build/images/perftool
-docker build -t antrea/perftool:latest .
+docker build -t antrea/perftool:latest -f Dockerfile.ubuntu .
+docker push antrea/perftool:latest
+```
+
+```bash (Windows)
+cd build/images/perftool
+docker build -t antrea/perftool:latest -f Dockerfile.windows .
 docker push antrea/perftool:latest
 ```
```

Comment:
are you sure this works? I mean that unless you use a proper multi-arch Docker manifest, I don't believe you can push 2 different Docker images (for 2 different archs) to the same registry with the same name & tag.

Reply:
Used different names and tags.